src/Microsoft.DotNet.Darc/src/DarcLib/VirtualMonoRepo/GitRepoClonerFactory.cs

```diff
@@ -22,25 +23,13 @@ public GitRepoClonerFactory(VmrRemoteConfiguration vmrRemoteConfig)
         _vmrRemoteConfig = vmrRemoteConfig;
     }
 
-    public IGitRepoCloner GetCloner(string repoUrl, ILogger logger)
+    public IGitRepoCloner GetCloner(string repoUri, ILogger logger) => GitRepoTypeParser.ParseFromUri(repoUri) switch
```

Comment:
Just a question - is it easy or difficult to see the return value used for the switch in this case when debugging?

Reply:
You can set a breakpoint on the block after `=>`:

![image](https://user-images.githubusercontent.com/7013027/206725130-680fb389-7b10-4b06-81dc-44f230378dfa.png)

Also the "Autos" window has the return value in VS (you can set breakpoints at the end of a method)